lib/browser/fingerprint.ts

```diff
@@ -72,8 +80,8 @@ export default function fingerprint(sdk: OktaAuthHttpInterface, options?: Finger
   return promise.finally(function() {
     clearTimeout(timeout);
     removeListener(window, 'message', listener);
-    if (document.body.contains(iframe)) {
-      iframe.parentElement.removeChild(iframe);
+    if (container.contains(iframe)) {
+      iframe.parentElement?.removeChild(iframe);
```

Comment:
What happens if `.parentElement.removeChild` is `undefined`. I assume it's possible since `?.` is being used